SimPEG/simulation.py

```diff
@@ -311,27 +468,35 @@ def make_synthetic_data(
         random_seed=None,
         **kwargs,
     ):
-        """
-        Make synthetic data given a model, and a standard deviation.
+        r"""Make synthetic data for the model and Gaussian noise provided.
+
+        This method generates and returns a :py:class:`SimPEG.data.SyntheticData` object
+        for the model and standard deviation of Gaussian noise provided.
 
         Parameters
         ----------
-        m : array
-            Array containing with geophysical model.
-        relative_error : float
-            Standard deviation.
-        noise_floor : float
-            Noise floor.
-        f : array or None
-            Fields for the given model (if pre-calculated).
+        m : (n_param, ) numpy.ndarray
+            The model parameters.
+        relative_error : float, SimPEG.data.UncertaintyArray
```

Comment:
```suggestion
        relative_error : float, numpy.ndarray
```
is a `SimPEG.data.UncertaintyArray` an object that we have? It might be something historical, but we should stick to standard array types here